js/signup.js

```diff
@@ -0,0 +1,74 @@
+const base_api = "https://floesia-api.herokuapp.com";
+
+window.onload = () => {
+    const signupBtn = document.querySelector("#signupBtn");
+    signupBtn.addEventListener("click", () => {
+        clearAlertBox();
+        signup();
+    })
+}
+
+function signup() {
+    const data = getFormData();
+    fetch( `${base_api}/signup`, genPostData(data) )
+        .then( data => {
+
+            console.log(data.status);
+
+            if (data.status === 404) {
+                redAlert('Incorrect nickname, email or password.')
+            } else {
+                data
+                    .json()
+                    .then(data => {
+                        storeToken(data.token);
```

Comment:
Just a quick request here. In the case of the signup, the token will not be returned (since the client still needs to make login). So the line ```storeToken(data.token);``` should be removed.

Reply:
this line will be removed in the next commit.
Thanks!